defaults/main.yml

```diff
@@ -15,6 +17,10 @@ postgresql_ctype_parts:
   - "UTF-8" # Encoding
 postgresql_ctype: "{{ postgresql_ctype_parts | join('.') }}"
 
+postgresql_env:
```

Comment:
Moved `postgresql_env` close to the other environment variables, rather than in some random place further down in the file.